### ISSUE TITLE
WCAG 4.1.2: ARIA hidden element must not contain focusable elements (.remove)

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2306]
#### Issue
`ARIA hidden element must not contain focusable elements` ([aria-hidden-focus](https://dequeuniversity.com/rules/axe/3.2/aria-hidden-focus?application=msftAI))
#### Target application
[Microsoft MakeCode for micro:bit](https://makecode.microbit.org/)
#### Element path
.remove
#### Snippet
`<i class="icon remove circle " aria-hidden="true" role="presentation" tabindex="0"></i>`
#### How to fix
    Fix all of the following:
      Focusable content should have tabindex='-1' or be removed from the DOM
#### Environment
Chrome version 75.0.3770.142
This accessibility issue was found using Accessibility Insights for Web 2.7.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

close button for expanded projects on home screen
